src/js/jsx/sections/libraries/assets/AssetPreviewImage.jsx

```diff
@@ -49,6 +49,15 @@ define(function (require, exports, module) {
          * @type {number}
          */
         _elementLastModified: 0,
+
+        /**
+         * Rendition update promise currently running for this asset preview
+         * will be cancelled if still around during unmount time
+         * This was added to prevent using `this.isMounted()`
+         *
+         * @type {Promise}
+         */
+        _renditionUpdate: Promise.resolve(),
```

Comment:
It's generally not kosher to define prototype properties using mutable values. I think you're being careful here not to modify the object directly on the prototype, but it would be safer to just initialize this null here and then in `componentWillUnmount` to see if the value is truthy before calling `cancel` on it.